### PR TITLE
retry mechanism for openshift install payload

### PR DESCRIPTION
**Pull request by Harshith-umesh**
### Description
extract the binary baseurl from openshift install binary url and poll the openshift server to come back to life 
needed update on pr : https://github.com/cloud-bulldozer/scale-ci-deploy/pull/55

@chaitanyaenr @jtaleric

### CODE REVIEW

File: OCP-4.X/roles/openshift-install/tasks/main.yml [hunk 1]
```diff
@@ -183,6 +183,23 @@
   when: platform == "gcp"
 
 - block:
+   - name: Check for install payload
+     shell: |
+       echo {{ openshift_install_binary_url }} | sed -e 's/\(.*\)openshift-install.*/\1/'
+     register: var
+
+   - name: print base url
```
Comment (chaitanyaenr) [inline OCP-4.X/roles/openshift-install/tasks/main.yml:191]: Maybe skip this step? ansible run with verbose output will display the output in the previous task. Thoughts?

Comment (Harshith-umesh) [reply OCP-4.X/roles/openshift-install/tasks/main.yml:191]: yes , can do that

**Review by chaitanyaenr — APPROVED** [review]
LGTM

**Pull request merged**